modules/react/tabs/lib/TabsList.tsx

```diff
@@ -124,36 +179,3 @@ export const TabsList = createSubcomponent('div')({
     );
   }
 );
-
-export const useTouchDirection = () => {
-  const [isDragging, setIsDragging] = React.useState(false);
-  const [touchDir, setTouchDirection] = React.useState<'right' | 'left' | undefined>('right');
-
-  React.useEffect(() => {
-    let prevXPos = window.pageXOffset;
-    const handleTouchMove = function (e: TouchEvent) {
-      const currXPos = e.touches[0].clientX;
-      setIsDragging(true);
-      if (currXPos > prevXPos) {
-        setTouchDirection('left');
-      } else if (currXPos < prevXPos) {
-        setTouchDirection('right');
-      }
-      prevXPos = currXPos;
-      e.preventDefault();
-    };
-    const handleDragEnd = function () {
-      setIsDragging(false);
-    };
-    window.addEventListener('touchmove', handleTouchMove);
-    window.addEventListener('touchstart', handleDragEnd);
-    window.addEventListener('touchend', handleDragEnd);
-    return () => {
-      window.removeEventListener('touchmove', handleTouchMove);
-      window.removeEventListener('touchstart', handleDragEnd);
-      window.removeEventListener('touchend', handleDragEnd);
-    };
-  }, []);
-
-  return {direction: touchDir, isDragging: isDragging};
-};
```

Comment:
Nothing is using this externally exported hook. It isn't doing anything particularly useful IMO. I don't see a use-case for indicating active scroll direction for this fade-effect.